### ISSUE TITLE
Remote Hour of finance - how to enable?

### ISSUE THREAD

**Issue by sarupbanskota** [sarupbanskota/social-event-ideas#2]
Hola folks, I hope everybody's weekend is kickass.

In the past week, we had a lot of commentary within the Canopy product dev team about organising social events remotely, and we ended up with a list [here](https://github.com/sarupbanskota/social-event-ideas/issues/1) (feel free to contribute).

One of the ideas that has gathered interest is the  [_Hour of Finance_](https://github.com/sarupbanskota/social-event-ideas/issues/1#issuecomment-306737668). I originally meant it to be a fork of [_Hour of Code_](https://hourofcode.com). I think finance, just like code, is an interesting domain that affects all of us quite deeply, yet it remains obscure to most of us. 

Was curious how could we better enable finance education amongst us, so that more people have access to the basic ideas, practices, tips etc around financey-thinks, like:
- stock markets
- property investments 
- foreign exchange 
- bitcoin
- robo advisory

Maybe it's worth doing a talk like `What would you recommend people do with their disposable money / savings when they're X years old?` Of course that answer would vary person to person, but it would be cool to see what people in the finance world themselves do with their personal savings. For a lot of us, we take the simple route of opening a Fixed Deposit in the bank or getting involved in some Bank investment scheme.

**Comment by rajnish4unow** [comment]
I think it's good to know what each of us is using. I use below for investment
Mutual Funds: 
 - fundsindia.com (Human Advice)
 - scripbox.com (Robo Advisory)

Equity Market
 - sharekhan.com
 - fundsindia.com
 
property investments

**Comment by fhlee** [comment]
Sharing something I wrote some time back (writing is another one of my hobby):
~~~~~
FIXED DEPOSIT (Savings): I am seriously against over-stretching ourselves with loans/debt. But I also believe it is wrong to "over-save".

There are many "savers" out there who don't invest our money. Due primarily to Fear-of-Loss, we will often just put our money into Fixed Deposit.
~~ "It is Safe, and it is Guaranteed." ~~

FALSE: This statement is far from the truth. Sure, investments do have risks. Sometimes (esp short-term) we may lose money. This is exactly why many of us prefer not to invest, and instead, choose to "multiply" our money with FD. ~ We are afraid to lose money. ~
When we invest, there is a POSSIBILITY that we may lose money. However, when we save (in FD), we are GUARANTEED to lose money.

HUH?? In this world, everything is relative --Sure, with FD, we will see our money "increase" over time; at 3~4% a year. "Slow, steady and guaranteed" one may say.
~~~~~
But, the truth is, FD is a GUARANTEED-LOSS strategy. 
~~~~~
WHY? Because there is no bank in this world that offers interest-rate higher than inflation --Read this again.
~~~~~
Sure enough, with FD our money seems to increase over time; BUT the value of our funds keeps depreciating over time.
Meaning: After 10 years of FD, although we will see a bigger-number, BUT the fact is that the Value of our money is actually reduced!!!
~~~~~
So, Fixed Deposit is a sure Slow-And-Steady way to LOSE money; and not the slow-and-steady way to Make money as most of us believe it is.

REALIZE THIS: So, for many of us, instead of taking a (some) risk on investments that could beat inflation; we choose FD --We opt to take a slow-and-guaranteed way to LOSE money instead. Not very smart isn't it? That is exactly what the banks want us to do.
~~~~~
After this, I hope we will look at FD from a different perspective.
We don't need to be involved with active-investments. There are many form of investments, some are more suitable to us than others. But trust me, FD is not one of them.
~~~~~
REMEMBER: If we do what 90% of the population does, we will end up being that 90%. To be the other 10%, we need to think differently.

**Comment by abhishekbose87** [comment]
@fhlee I agree with your thoughts made on `FixedDeposits`. It would be great if you can take a 30mins/ 60mins finance session to talk about investments which can beat inflation but at the same time do not require a lot of tracking or handholding.
